app/components/agendaitems/agendaitems-filter/template.hbs

```diff
@@ -26,9 +26,19 @@
         </div>
       </div>
     </div>
-    <div class="vl-col--4-12 vl-col--1-1--s">
+    <div class="vlc-input-field-block vl-col--4-12">
       <label class="vl-form__label" for="agendaItemId">
         {{t "agendaitem-field-label"}}
+        <span class="vl-icon vl-icon--small vl-vi vl-vi-info-filled vlc-u-cursor-pointer" aria-hidden="true">
+          {{#attach-popover
+            placement="right"
+            showOn="click"
+            hideOn="click"
+            class="ember-attacher-popper vlc-search-legend-popover"
+          }}
+            {{utils/search-legend}}
+          {{/attach-popover}}
```

Comment:
We totally forgot that this component will also need to have the `isShown` logic used in `cases-filter`. Atm the close button doesn't work there, and you also can't close it using the clickout.

Reply:
I've updated this modal with everything that we applied on the `cases-filter` component